lib/Backend/GlobOptBailOut.cpp

```diff
@@ -916,7 +916,7 @@ GlobOpt::MayNeedBailOnImplicitCall(IR::Opnd * opnd, Value *val, bool callsToPrim
     case IR::OpndKindReg:
         // Only need implicit call if the operation will call ToPrimitive and we haven't prove
         // that it is already a primitive
-        return callsToPrimitive &&
```

Comment:
The parameter is just useless now. Every opcode that gets here will have OpndHasImplicitCall (which subsumes OpCallsValueOf) set.